### PR TITLE
Migrating the Privacy Fragment to constraint layout

### LINKED ISSUE #81: [Enhancement] Migrating layouts to constraint layout
- Need to migrate maximum layouts to constraint layouts.
- [Why to migrate to constraint layouts?](https://android-developers.googleblog.com/2017/08/understanding-performance-benefits-of.html)
- Please assign me this issue under hacktoberfest.

- [x] HomeFragment
- [ ] ProfileFragment
- [x] NearByFragment
- [x] SignupFragment
- [ ] SignInFragment
- [ ] MessageFragment
- [ ] EditProfileFragment
- [x] PrivacyFragment 
- [ ] ViewProfileFragment
- [ ] VideoCallFragment
- [x] NotificationFragment

### PR DESCRIPTION
**Pull request by AdarshThakare**
## Description
I successfully migrated the UI of the Privacy Screen from Frame Layout to Constraint layout

fixes #81 

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

 ## Screenshot of successful build.
Screenshot 2024-10-11 at 6 53 15 PM

## Screenshots / Screen Recordings
Screenshot 2024-10-11 at 6 47 20 PM
Screenshot 2024-10-11 at 6 46 10 PM

### CODE REVIEW

File: app/src/main/res/layout/activity_privacy.xml [hunk 1]
```diff
@@ -5,9 +5,29 @@
...
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/privacy_recycler_view"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintTop_toBottomOf="@id/appbar"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent" />
+
 </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (Vishwajith-Shettigar) [inline app/src/main/res/layout/activity_privacy.xml]: Add extra line in EOF.

File: app/src/main/res/layout/activity_privacy.xml [hunk 2]
```diff
@@ -5,9 +5,29 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".privacy.PrivacyActivity">
-    <FrameLayout
-        android:id="@+id/frame_layout"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        />
+    <!-- AppBarLayout with Toolbar -->
```
Comment (Vishwajith-Shettigar) [inline app/src/main/res/layout/activity_privacy.xml]: Remove this comment.

File: app/src/main/res/layout/activity_privacy.xml [hunk 3]
```diff
@@ -5,9 +5,29 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent">
+
+        <androidx.appcompat.widget.Toolbar
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            app:title="Privacy" />
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <!-- RecyclerView for Privacy Items below the AppBarLayout -->
```
Comment (Vishwajith-Shettigar) [inline app/src/main/res/layout/activity_privacy.xml]: Remove this comment.

**Review by Vishwajith-Shettigar — CHANGES_REQUESTED** [review]

**Comment by AdarshThakare** [comment]
made the changes that u requested under review

**Review by Vishwajith-Shettigar — APPROVED** [review]

**Comment by Vishwajith-Shettigar** [comment]
Thanks @AdarshThakare

**Comment by AdarshThakare** [comment]
Thanks a lot @Vishwajith-Shettigar . Happy to help

**Pull request merged**